src/types/store/productStateTypes.ts

```diff
@@ -11,4 +12,5 @@ export interface ProductsState extends IState {
 
 export interface ProductState extends IState {
   product: null | IProduct;
+  additionalInfo: null | IAdditionalInfo;
```

Comment:
It's better to have IAdditionalInfo | null, type is first

Reply:
Swaped types for null to come last.